src/numbers/uint128.js

```diff
@@ -0,0 +1,17 @@
+import { LargeInt } from 'js-xdr';
+
+export class Uint128 extends LargeInt {
+  constructor(...args) {
```

Comment:
this will inherit the same constructor from base class, and skip re-declaring here in the child classes..I think.

Reply:
Yup, this should inherit the base class constructor automatically.

But doesn't hurt to double check that this still works if you remove it 😅

Reply:
yep I like that it's explicit bc it will document, added jsdocs to make this even more obvious to users